builders/integrations/oracles/api3.md

```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
-
-You can then decide if you want to use self-funded or managed dAPIs.
-
-![API3 Dapi Page](/images/builders/integrations/oracles/api3/api3-5.webp)
-
-#### Fund a Sponsor Wallet {: #fund-sponsor-wallet }
-
-Once you have selected your dAPI, you can activate it by using the [API3 Market](https://market.api3.org){target=\_blank} to send funds (DEV, MOVR, or GLMR) to the `sponsorWallet`. Make sure your:
-
-- Wallet is connected to the Market and is the same network as the dAPI you are funding
-- The balance of your wallet should be greater than the amount you are sending to the `sponsorWallet`
-
-To fund the dAPI, you need to click on the **Fund Gas** button. Depending upon if a proxy contract is already deployed, you will see a different UI.
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-6.webp)
-
-Use the gas estimator to select how much gas is needed by the dAPI. Click on **Send DEV** to send the entered amount to the sponsor wallet of the respective dAPI.
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-7.webp)
-
-Once the transaction is broadcasted & confirmed on the blockchain a transaction confirmation screen will appear.
+| Deviation | Heartbeat |
+| --------- | --------- |
+| 0.25%     | 24 hours  |
+| 0.5%      | 24 hours  |
+| 1%        | 24 hours  |
+| 5%        | 24 hours  |
 
-#### Deploy a Proxy Contract to Access The dAPI {: #deploy-proxy }
+![API3 Market 1](/images/builders/integrations/oracles/api3/dapi-1.png)
 
-Smart contracts can interact and read values from contracts that are already deployed on the blockchain. By deploying a proxy contract via the API3 Market, a dApp can interact and read values from a dAPI like ETH/USD.
+#### Activating your dAPI {: #select-a-dapi }
 
 !!! note
-    If a proxy is already deployed for a self-funded dAPI, the dApp can read the dAPI without having to deploy a proxy contract. They do this by using the address of the already deployed proxy contract which will be visible on the API3 Market.
-
-If you are deploying a proxy contract during the funding process, clicking on the **Get proxy** button will initiate a transaction to your wallet that will deploy a proxy contract.
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-8.webp)
-
-Once the transaction is broadcasted & confirmed on the blockchain, the proxy contract address will be shown on the UI.
+    If a dAPI is already activated, make sure to check the expiration date and update parameters. You can update the parameters and extend the subscription by purchasing a new configuration.
```

Comment:
How can I tell if the dAPI is already activated?

Reply:
I was checking out the testnet ETH/USD feed, and it showed the following:

<img width="466" alt="Screenshot 2024-06-11 at 10 13 39 PM" src="https://github.com/moonbeam-foundation/moonbeam-docs/assets/26533957/0c71ed77-ed67-479b-86f1-7bf317f9abfc">

Then I clicked purchase new plan, connected my wallet, clicked pruchase ETH/USD and signed the transaction. The transaction successfully went through, but nothing really changed on the UI. It shows the same as the above screenshot.

Reply:
You purchased the dAPI for the same parameters. You can purchase a better configuration for it. (For example: 0.25% deviation and 24hr heartbeat).

Reply:
Haha ah ok... makes sense 😂 thanks!